lib/tbot/service_workload_identity_aws_ra.go

```diff
@@ -91,7 +91,16 @@ func (s *WorkloadIdentityAWSRAService) generate(ctx context.Context) error {
 	if err != nil {
 		return trace.Wrap(err, "marshaling private key")
 	}
-	svid, err := x509svid.ParseRaw(res.GetX509Svid().Cert, pkcs8)
+	certWithChain := bytes.NewBuffer(res.GetX509Svid().Cert)
```

Comment:
Avoid mutating unrelated data (the `[]byte` passed to `NewBuffer` is actually used as the backing store until it needs to be resized), prefer getters when dealing with protobuf messages.

```suggestion
	certWithChain := new(bytes.Buffer)
	buffer.Write(res.GetX509Svid().GetCert())
```

Reply:
d1d9285d21